### PR TITLE
Create Evelinalunga

### PR DESCRIPTION
**Pull request by Evelinalunga**

### CODE REVIEW

**Comment by Evelinalunga** [comment]
@stefanbuda

File: bios/Evelinalunga [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hello, just wanted to ask if there will also be a written user guide for Githube.
+Would be really usefull
```
Comment (Evelinalunga) [inline bios/Evelinalunga:2]: test

**Comment by githubteacher** [comment]
@Evelinalunga we do have excellent documentation on GitHub.com! guides.github.com is one example!

File: bios/Evelinalunga (hunk 1 above)
Comment (bdurrant) [inline bios/Evelinalunga:2]: :+1:

**Pull request merged**